specs/DesignDiscussions/InputLabels.md

```diff
@@ -1,3 +1,52 @@
+# Table of Contents
+1. [Input/Label Associations](#InputLabel-Associations)
+	1. [Basic Label Support in Adaptive Cards](#Basic-Label-Support-in-Adaptive-Cards) 
+	2. [Formatting](#Formatting)
+		1. [Default rendering for labels](#Default-rendering-for-labels)
+		2. [Required indicators](#Required-indicators) 
+		3. [Card Author Formatting](#Card-Author-Formatting)
+			1. [Option 1 – Allow TextBlocks and RichTextBlocks as Labels](#Option-1-–-Allow-TextBlocks-and-RichTextBlocks-as-Labels)
+			2. [Option 2 – LabelFor Property on TextBlocks/RichTextBlocks]([#Option-2-–-LabelFor-Property-on-TextBlocks/RichTextBlocks])
+			3. [Final Conclussion](#Final-conclussion)
+	3. [Backwards Compatibility](#Backwards-Compatibility)
+	4. [Placeholders](#Placeholders)
+	5. [Label Position](#Label-Position)
+	6. [Other Input Types](#Other-Input-Types)
+		1. [Date Input](#Date-Input)
+		2. [Time Input](#Time-Input)
+		3. [Number Input](#Number-Input)
+		4. [Choice Set](#Choice-Set)
+			1. [Expanded](#Expanded)
+			2. [Compact](#Compact)
+		5. [Toggle Inputs](#Toggle-Inputs)
+	7. [Feature Cost Estimation](#Feature-Cost-Estimation)
+		1. [Follow ups](#Follow-ups) 
+	8. [Accessibility](#Accessibility)
+		1. [1.3.1 Info and Relationships](#131-Info-and-Relationships)
+		2. [2.4.6 Headings and Labels](#246-Headings-and-Labels)
+		3. [3.3.2 Labels and Instructions](#332-Labels-and-Instructions)
+2. [Appendix](#Appendix)
```

Comment:
We should also annotate requirements wiht Priorities and explain the mapping in the Appendix: (Here's a stab at this  - @matthidinger / @dclaux let me know your thoughts)
P0 - Required feature included in v1 feature release; Counts towards Accessibility and/or Teams MVP scenario
P1 - Good to have feature that should be considered due to feature completeness/Survey demand
P2 - Nice to have feature that has been demanded via customer surveys, but can be included in a follow up release.